behavior in multi instances environment

mmisecius opened the pull request:
corrected 'events-list' selector problem,  clicking and double clicking problem on the chosen day, all-day event problem

Serhioromano commented on js/calendar.js:
```diff
@@ -454,8 +454,8 @@ if(!String.prototype.formatNum) {
 		var end = new Date(this.options.position.end.getTime());
-		end.setHours(time_end[0]);
-		end.setMinutes(time_end[1]);
+		//end.setHours(time_end[0]);
```
Why these lines are commented?

mmisecius replied:
yep,

I have commented out those lines because 'all day event' has not appeared
properly - I think there is problem with setting of the border of the of
period. After I commented it out - it seemed to start working correctly -
but you can verify this.





2015-06-08 7:41 GMT+02:00 Sergey Romanov <notifications@github.com>:

> In js/calendar.js
> <https://github.com/Serhioromano/bootstrap-calendar/pull/491#discussion_r31887991>
> :
>
> > @@ -454,8 +454,8 @@ if(!String.prototype.formatNum) {
> >  		start.setHours(time_start[0]);
> >  		start.setMinutes(time_start[1]);
> >  		var end = new Date(this.options.position.end.getTime());
> > -		end.setHours(time_end[0]);
> > -		end.setMinutes(time_end[1]);
> > +		//end.setHours(time_end[0]);
>
> Why these lines are commented?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Serhioromano/bootstrap-calendar/pull/491/files#r31887991>
> .
>

Serhioromano commented:
Could you tell more what was the problem?

mmisecius commented:
about multi  instances problem?  If you have more than one calendar
instances on the same page (e.g in various tabs) there is problem of jquery
selector - so that I added 'instanceId' into several calling of jquery
'get' method and i seems it works now. in our project we currently use
modified version of your library and everything work as excepted. by the
way  your calendar look nice and can be customize nicely (day, week,
template etc)

thanks

m.

2015-06-08 7:41 GMT+02:00 Sergey Romanov <notifications@github.com>:

> Could you tell more what was the problem?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Serhioromano/bootstrap-calendar/pull/491#issuecomment-109870575>
> .
>

Serhioromano commented on js/calendar.js:
```diff
@@ -1089,8 +1090,8 @@ if(!String.prototype.formatNum) {
-
-		$('.cal-month-day, .cal-year-box .span3')
+		var calendarSelector = self.context.selector;
+		$(calendarSelector + ' .cal-month-day, .cal-year-box .span3')
```
Would it be better to use context like this

    $('.cal-month-day, .cal-year-box .span3', self.context)

Then it will look inside calendar. And it is cleaner. Also your way it should be

    $(calendarSelector + ' .cal-month-day, ' + calendarSelector + '.cal-year-box .span3')
